core/scss/elements/input/_input.scss

```diff
@@ -74,7 +74,7 @@ fieldset {
 [type=radio] {
   // The actual input element is only visible to screen readers, because
   // all visual styling is done via the label.
-  @include sr-only;
+  @include hide-visually;
```

Comment:
Really? All checkboxes and radio buttons are hidden by default?

Reply:
This component has not been touched for a very long time, and includes (mostly) code from the U.S. Web Design System (https://github.com/uswds/uswds). Currently, the visual styling for these are being done on the label, which is what the input element itself is "hidden." Nonetheless, I think we'll want to revisit this and make it mo betta overall.

Because I dropped the `sr-only` mixin in favor of the`hide-visually` and `hide-text` Bourbon mixins, the compiler was throwing an error ("...cannot find `sr-only` mixin..."). So, I quickly replaced it, which is why it is appearing as part of this PR.

Reply:
Roger that. Thanks for the explanation. I do think we'll want to revisit this, but not as part of this PR.